pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -308,6 +301,12 @@ def init_ddp_connection(self, global_rank: Optional[int] = None, world_size: Opt
             log.info(f"initializing ddp: GLOBAL_RANK: {global_rank}, MEMBER: {global_rank + 1}/{world_size}")
             torch_distrib.init_process_group(self.torch_distributed_backend, rank=global_rank, world_size=world_size)
 
+            # on rank=0 let everyone know training is starting
+            rank_zero_info("-" * 100)
+            rank_zero_info(f"distributed_backend={self.distributed_backend}")
+            rank_zero_info(f"All DDP processes registered. Starting ddp with {self.world_size} processes")
+            rank_zero_info("-" * 100)
```

Comment:
can we rather form it as one long message instead of 4x calls?

Reply:
can you show how?
the string concatenation with line breaks only works when the whole line is a string, but here we have a `* 100` int at the end.